src/us/kbase/workspace/database/provenance/Common.java

```diff
@@ -4,19 +4,24 @@
 import static us.kbase.workspace.database.Util.isNullOrEmpty;
 import static us.kbase.workspace.database.Util.noNulls;
 
+import static java.util.Objects.requireNonNull;
+
 import java.net.MalformedURLException;
 import java.net.URISyntaxException;
 import java.net.URL;
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 /* This is an internal only class for shared code between the provenance classes.
  * Tests are all via testing the public provenance class APIs.
  */
 class Common {
-	
-	private Common() {};
+
```

Comment:
I ran Eclipse's "format" command to fix up the code I'd added, so there are whitespace changes in addition to the new code.

Reply:
Update: undid the format command.